integration-tests/scripts/konflux-e2e/0.2/konflux-e2e.sh

```diff
@@ -18,6 +18,7 @@ load_envs() {
     declare -A load_envs_from_file=(
         [DEFAULT_QUAY_ORG_TOKEN]="${konflux_ci_secrets_file}/default-quay-org-token"
         [QUAY_TOKEN]="${konflux_ci_secrets_file}/quay-token"
+        [RELEASE_CATALOG_TA_QUAY_TOKEN]="${konflux_ci_secrets_file}/release-catalog-ta-quay-token"
```

Comment:
We also need to add this line to "integration-tests/scripts/konflux-e2e/0.1/konflux-e2e.sh" file.

Reply:
good catch. thanks! fixed